### DISCUSSION TITLE
Updating to latest version

### DISCUSSION THREAD

**Discussion by bartcontrol** [python-control/python-control#979] [Q&A]
Dear all,

I would like to update to the latest version, I am now running 0.9.4 however I would like to use some MIMO possibilities for plotting the bode plots and step responses. I now get the error: AttributeError: 'TimeResponseData' object has no attribute 'plot'

Which I think is because I have not updated to the latest version, since I cannot find this anywhere in the 0.9.4 docs.

However when I try to update using conda update, it tells me I have the latest version, which is 0.9.4 if i use conda list to show my packages and versions.

Can anyone help me with this?

Thanks!

**Comment by murrayrm** [comment]
The latest official release is v0.9.04.

The current development code (which will become v0.10.0) is only available via GitHub, not conda.

I do hope to post v0.10.0 in the coming week (just finishing off a PR to update the phase plot module).  So if you wait for a bit you can get the latest version via conda (and PyPI).

**Reply by bartcontrol** [reply]
Thank you for your response! I look forward to start working with the new version!

**Comment by murrayrm** [comment]
Version 0.10.0 is now released.  It should be available now on PIPy and soon on conda-forge.
